src/kibana/plugins/kibana/_timepicker.js

```diff
@@ -24,7 +24,9 @@ define(function (require) {
       // Close if already open
       if ($scope.globalConfigTemplate === timepickerHtml) {
         delete $scope.globalConfigTemplate;
+        document.getElementById('timepicker').setAttribute('aria-expanded', 'false');
```

Comment:
It appears to me you wrote the necessary code for this in the your latest commit, now this just have to be removed.